services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java

```diff
@@ -245,11 +245,11 @@ public void closePreparedStatement(
     try {
       preparedStatementHandle =
```

Comment:
No need to declare this variable outside the try block now.

Reply:
Fixed